### PR TITLE
Homepage Updates

### PR DESCRIPTION
**Pull request by spbyrne**
Work in progress

### CODE REVIEW

File: content/pages/cloud.json [hunk 1]
```diff
@@ -80,56 +80,16 @@
...
-          "subline": "User-management, auth, roles, and a GraphQL API for your content",
-          "icon": "",
-          "featured": "true",
-          "media": {
-            "src": "img/headlessCms.png"
-          }
-        }
-      ]
+      "headline": "*Edit Content, Not Code*",
+      "subline": "Make changes in contenxt and see them in in real time. Hit save and Tina handles the commit.",
```
Comment (DirtyF) [inline content/pages/cloud.json:89]:
```suggestion
      "subline": "Make changes in context and see them in real-time. Hit save and Tina handles the commit.",
```

File: content/pages/home.json [hunk 2]
```diff
@@ -55,87 +55,47 @@
...
-            "src": "/img/visual-editing.gif"
+            "src": "/img/edit-alongside-content.gif"
           }
         },
         {
           "_template": "feature",
-          "headline": "Your Design System & Components",
-          "subline": "Allow your team to pick from your custom, predefined components to build pages.  Maintain a simpler codebase that offers more versatility.",
+          "headline": "Built for the Jamstack",
+          "subline": "Tina is designed for Next.js sites but we plan to support other frameworks.",
```
Comment (DirtyF) [inline content/pages/home.json:69]:
```suggestion
          "subline": "Tina is extensible by design, and already supports React and Next.js based websites.",
```

File: content/pages/home.json [hunk 3]
```diff
@@ -55,87 +55,47 @@
...
-            "src": "/img/jamstack.png"
+            "src": "/img/your-blocks.gif"
           }
         },
         {
           "_template": "feature",
-          "headline": "Git for the Whole Team",
-          "subline": "Tina supports content stored in Git. Give the benefits of branching and version-control to the whole team.",
+          "headline": "Content Backed by Git",
+          "subline": "Tina supports content stored in Git. Give the benefits of version-control to the whole team.",
```
Comment (DirtyF) [inline content/pages/home.json:85]:
```suggestion
          "subline": "Tina supports content stored in Git. Give the benefits of version-control to your whole team.",
```

File: content/pages/home.json [hunk 4]
```diff
@@ -55,87 +55,47 @@
...
-          "headline": "3rd Party API",
-          "subline": "Store your content in Airtable, Google Sheets, or any data source",
-          "icon": "arrowRight",
-          "media": {
-            "src": "/img/headlessCms.png"
-          }
-        }
-      ]
+      "headline": "*Edit Content, Not Code*",
+      "subline": "Make changes in contenxt and see them in in real time. Hit save and Tina handles the commit.",
```
Comment (DirtyF) [inline content/pages/home.json:95]:
```suggestion
      "subline": "Make changes in context and see them in real-time. Hit save and Tina handles the commit.",
```

File: content/pages/cloud.json [hunk 5]
```diff
@@ -62,7 +62,7 @@
           "headline": "Move Faster With Tina Cloud",
           "subline": "Visual editing + Git storage allows you to ship content faster with the safety of version-control.",
```
Comment (DirtyF) [inline content/pages/cloud.json:63]:
```suggestion
          "subline": "Visual editing and Git storage allows you to ship content faster with the safety of version-control.",
```

**Comment by spbyrne** [comment]
Thank you @DirtyF !

**Review by DirtyF — APPROVED** [review]

**Pull request merged**